<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgHome</class>
 <widget class="QDialog" name="DlgHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统复位</string>
  </property>
  <widget class="QLabel" name="label_home_notice">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>正在进行机台复位操作，请确定是否需要复位机台？</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_hometip">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_homept">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>平台</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_home_exit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>离开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_homeall">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>全机</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
